cds_ils/templates/invenio_app_ils_circulation/mail/cds_active_loans.html

```diff
@@ -5,7 +5,7 @@
 {% block body_plain %}
 Dear librarians,
 
-when removing users who are no longer registered in the system, it has been found a patron with active loans.
+when removing users who are no longer registered in the system, a patron with active loans has been found.
```

Comment:
I have completely rewritten this. If you remove it from your PR, you won't have a conflict.